recipes/FC-Virus/mete.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: FC-Virus
+  version: "1.0.0"
+  noarch: generic
+
+source:
+  url: https://github.com/qdu-bioinfo/FC-Virus/archive/refs/heads/main.tar.gz
+  git_tag: main
+  sha256: a5c91ea315dc7b64b75986128d97c08522aae332cb640fd2c115cc379b4adc51
+
+build:
+  number: 0
+  skip: true
+
+requirements:
+  build:
+    - gcc
+    - gxx_linux-64
+    - make
```

Comment:
```suggestion
    - make
  host:
```

Reply:
Thank you for your reply. I have completed the modifications.